source/util/hex_float.h

```diff
@@ -76,6 +76,14 @@ struct FloatProxyTraits<float> {
   static float max() { return std::numeric_limits<float>::max(); }
   // Returns the lowest normal value.
   static float lowest() { return std::numeric_limits<float>::lowest(); }
+  // Returns the value as the native floating point format.
+  static float getAsFloat(const uint_type& t) {
+    return BitwiseCast<float>(t);
+  }
+  // Returns the value as the native floating point format.
```

Comment:
Method comment is copypasta

Reply:
While we're here, maybe rename the method to getBitsFromFloat  ?
That better mirrors the FloatProxy::getBits name.